server/user.go

```diff
@@ -59,7 +59,7 @@ func httpUserConnect(ji Instance, w http.ResponseWriter, r *http.Request) (int,
 
 	// Users shouldn't be able to make multiple connections.
 	if jiraUser, err := ji.GetPlugin().userStore.LoadJIRAUser(ji, mattermostUserId); err == nil && len(jiraUser.Key()) != 0 {
```

Comment:
While we're touching this code, I think we should probably refactor to our style of not using the `if err := ...; err != nil {` go idiom (just like lines 104 and 105 in `command.go` above).

Reply:
@cpoile I've updated the PR to reflect this